EvidenceApi.Tests/V1/UseCase/CreateAuditUseCaseTests.cs

```diff
@@ -0,0 +1,70 @@
+using System;
+using AutoFixture;
+using EvidenceApi.V1.Boundary.Request;
+using EvidenceApi.V1.Domain;
+using EvidenceApi.V1.Gateways.Interfaces;
+using EvidenceApi.V1.UseCase;
+using FluentAssertions;
+using Moq;
+using NUnit.Framework;
+
+namespace EvidenceApi.Tests.V1.UseCase
+{
+    public class CreateAuditUseCaseTests
+    {
+        private CreateAuditUseCase _classUnderTest;
+        private Mock<IEvidenceGateway> _evidenceGateway;
+        private readonly IFixture _fixture = new Fixture();
+        private AuditEventRequest _request;
+        private AuditEvent _created;
+
+        [SetUp]
+        public void SetUp()
+        {
+            _evidenceGateway = new Mock<IEvidenceGateway>();
+            _classUnderTest = new CreateAuditUseCase(_evidenceGateway.Object);
+
+        }
+
+        [Test]
+        public void ReturnsTheCreatedAuditEvent()
+        {
+            SetupMocks();
+            var result = _classUnderTest.Execute(_request);
+            Console.WriteLine(_request.Path);
+            Console.WriteLine(_created.UrlVisited);
+            Console.WriteLine(result.UrlVisited);
```

Comment:
Do we need to print these values to the console?

Reply:
No, going to remove them